detection_rules/schemas/registry_package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+"""Definitions for packages destined for the registry."""
+
+import dataclasses
+from typing import Dict, Union, Type
+
+import marshmallow_dataclass
+from marshmallow import validate
+
+from .definitions import BaseMarshmallowDataclass, ConditionSemVer, SemVer
+
+
+@marshmallow_dataclass.dataclass
+class BaseManifest(BaseMarshmallowDataclass):
+    """Base class for registry packages."""
+
+    conditions: Dict[str, ConditionSemVer]
+    version: SemVer
+    format_version: SemVer
+
+    categories: list = dataclasses.field(default_factory=lambda: ['security'].copy())
+    description: str = 'Rules for the detection engine in the Security application.'
+    icons: list = dataclasses.field(default_factory=list)
```

Comment:
basically, we're gonna make it an integration, but i'll be internal so it doesn't show up on Fleet's page.
```suggestion
    icons: list = dataclasses.field(default_factory=list)
    internal: bool = True
```